checkQC/handlers/unidentified_index_handler.py

```diff
@@ -74,13 +74,15 @@ def check_qc(self):
             for index in indices:
                 tag = index['index']
                 count = index['count']
-                # Unknown index means that the sample was run without an index. We
                 if self.should_be_evaluated(tag, count, number_of_reads_on_lane):
-                    yield from self.evaluate_index_rules(tag, lane, samplesheet_dict)
+                    percent_on_lane = float(count) / number_of_reads_on_lane
```

Comment:
Won't this be ratio rather than percent?

Reply:
Well, spotted. Fixed this now.